cmd/lakectl/cmd/annotate.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const annotateTemplate = `{{  $val := .Commit}}
+{{ $.Object|ljust 15}} {{ $val.Committer|ljust 20 }} {{ $val.Id | printf "%.16s"|ljust 20 }} {{ $val.CreationDate|date }}  {{ $.CommitMessage |ljust 30 }}
+`
+const messageSize = 100
```

Comment:
```suggestion
const messageSize = 200
```
Most people nowadays have terminals that are >100 chars wide.  Let's use that!

Reply:
Done.